src/DotNetBridge/DotNetBridge.csproj

```diff
@@ -32,21 +32,21 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.ML" Version="1.5.0-preview2" />
-    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.5.0-preview2" />
-    <PackageReference Include="Microsoft.ML.EntryPoints" Version="0.17.0-preview2" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.5.0-preview2" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.5.0-preview2" />
-    <PackageReference Include="Microsoft.ML.LightGBM" Version="1.5.0-preview2" />
-    <PackageReference Include="Microsoft.ML.OnnxConverter" Version="0.17.0-preview2" />
-    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.5.0-preview2" />
+    <PackageReference Include="Microsoft.ML" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.EntryPoints" Version="0.17.0" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.LightGBM" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.OnnxConverter" Version="0.17.0" />
+    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.5.0" />
     <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.2.0" />
```

Comment:
nit: We're using version 1.3.0 of onnx runtime:
https://github.com/dotnet/machinelearning/pull/5104

Reply:
thx

---
In reply to: [437769599](https://github.com/microsoft/NimbusML/pull/492#discussion_r437769599) [](ancestors = 437769599)